data/mods/Magiclysm/worldgen/black_dragon_lair.json

```diff
@@ -387,6 +387,8 @@
         ".": [ { "item": "lair_black_dragon", "chance": 15, "repeat": [ 0, 2 ] } ],
         " ": [
           { "item": "lair_black_dragon", "chance": 10, "repeat": [ 0, 2 ] },
+          { "item": "magic_weapons_loot_1", "chance": 15, "repeat": [ 0, 1 ] },
+          { "item": "magic_weapons_loot_2", "chance": 15, "repeat": [ 0, 1 ] },
```

Comment:
`magic_weapons_loot_2` is not defined anywhere and should be either removed or defined.